docs/README.Fedora.md

```diff
@@ -1,76 +1,108 @@
-jre gawk gperf java-9-openjdk-headless libao-devel alsa-lib-devel \
-libass-devel libva-devel avahi-devel avahi-compat-libdns_sd-devel bluez-libs-devel \
-libbluray-devel bzip2-devel libcap-devel \
-libcdio-devel libcec-devel openssl-libs \
-dbus-devel mesa-libEGL-devel fmt-devel fontconfig-devel freetype-devel \
-fribidi-devel giflib-devel \
-libjpeg-turbo-devel libtool-ltdl-devel lzo-devel libmicrohttpd-devel \
-libmpc-devel mariadb-devel libnfs-devel \
-pcre-devel libplist-devel libpng12-devel \
-shairplay-devel libsmbclient-devel sqlite-devel libssh-devel openssl-devel ffmpeg-devel \
-taglib-devel tinyxml-devel libtool-ltdl-devel libudev-devel \
-libusb-devel libva-devel libvdpau-devel libxml2-devel \
-libXmu-devel libXrandr-devel libxslt-devel libXt-devel rapidjson-devel \
-nasm python-devel yasm swig uuid-devel zlib-devel python-pillow \
-trousers-devel libidn2-devel
-
-
------------------------------------------------------------------------------
-4. How to compile
------------------------------------------------------------------------------
-See README.linux
-
------------------------------------------------------------------------------
-4.1. Test Suite
------------------------------------------------------------------------------
-See README.linux
-
------------------------------------------------------------------------------
-5. Uninstalling
------------------------------------------------------------------------------
-See README.linux/Uninstalling for removing compiled versions of Kodi.
-
-EOF
+![Kodi Logo](resources/banner_slim.png)
+
+# Fedora build guide
+This guide has been tested with Fedora 27 1.6 x86_64. Please read it in full before you proceed to familiarize yourself with the build procedure.
+
+Several other distributions have **[specific build guides](README.md)** and a general **[Linux build guide](README.Linux.md)** is also available.
+
+## Table of Contents
+1. **[Document conventions](#1-document-conventions)**
+2. **[Get the source code](#2-get-the-source-code)**
+3. **[Install the required packages](#3-install-the-required-packages)**  
+  3.1. **[Build missing dependencies](#31-build-missing-dependencies)**
+4. **[Build Kodi](#4-build-kodi)**
+
+## 1. Document conventions
+This guide assumes you are using `terminal`, also known as `console`, `command-line` or simply `cli`. Commands need to be run at the terminal, one at a time and in the provided order.
+
+This is a comment that provides context:
+```
+this is a command
+this is another command
+and yet another one
+```
+
+**Example:** Clone Kodi's current master branch:
+```
+git clone https://github.com/xbmc/xbmc kodi
+```
+
+Commands that contain strings enclosed in angle brackets denote something you need to change to suit your needs.
+```
+git clone -b <branch-name> https://github.com/xbmc/xbmc kodi
+```
+
+**Example:** Clone Kodi's current Krypton branch:
+```
+git clone -b Krypton https://github.com/xbmc/xbmc kodi
```

Comment:
why mention Krypton in master branch?

Reply:
It's the latest branch not master.

Reply:
So you will updated this all the time to not let user think, I should clone Krypton, because it is mentioned here?

Reply:
I agree, docs in master shouldn't reference other branches. You can add a note that this will not get you the current stable release

Reply:
It's master now. It will become Leia branch in no time. And no, I have no intention to continuously update this.

It's an **example** and clearly marked as such. That said, please decide appropriate course of action. I'm not changing 10 build guides just because.